bin/format.jl

```diff
@@ -81,3 +81,4 @@ if isempty(ARGS) || haskey(opts, :help)
     exit(0)
 end
 format(ARGS; opts...)
+
```

Comment:
You're going to want to get rid of these trailing newlines too, I'd guess (even though I think they're technically ok).